stack

```diff
@@ -95,20 +122,38 @@ function build-faucet() {
 # run the faucet container
 function faucet() {
   eval $(./stack print-contract-env)
+  echo $ADMIN_PRIVATE_KEY
   local privateKey=$(echo -n "$ADMIN_PRIVATE_KEY" | sed 's/^0x//')
+  echo $WEB3_TOKEN_ADDRESS
   local tokenAddress=$(echo -n "$WEB3_TOKEN_ADDRESS" | sed 's/^0x//')
-  docker run -d \
-    --name faucet \
-    -p 8085:8080 \
-    --restart always \
-    --network "$GETH_DOCKER_NETWORK" \
-    -e WEB3_PROVIDER=http://geth:8545 \
-    -e PRIVATE_KEY=$privateKey \
-    -e TOKEN_ADDRESS=$WEB3_TOKEN_ADDRESS \
-    $FAUCET_DOCKER_IMAGE \
-      --faucet.amount=100 \
-      --faucet.tokenamount=100 \
-      --faucet.minutes=1
+  echo $GETH_DOCKER_NETWORK
+  if [[ -n $LOCAL_GETH ]]; then
```

Comment:
maybe set a varaible for WEB3_PROVIDER in the if statement rather than copy pasting the entire docker command